client/src/ui/BitbakeCommands.ts

```diff
@@ -30,6 +30,11 @@ async function parseAllrecipes (bitbakeWorkspace: BitbakeWorkspace, taskProvider
     'Parse all recipes',
     'bitbake'
   )
+  const runningTasks = vscode.tasks.taskExecutions
```

Comment:
I noticed that ProjectScanner has a similar behaviour, do we want to keep that the same as here?
![image](https://github.com/savoirfairelinux/vscode-bitbake/assets/47988425/db182f72-949c-4eab-a0ac-89ddf343ce1d)

Reply:
Yes we can register and enqueue a new parsing. Added a commit for that.